pkg/scyllaclient/client_scylla.go

```diff
@@ -551,7 +553,7 @@ func dumpRanges(ranges []TokenRange) string {
 			_ = buf.WriteByte(',')
 		}
 		if ttr.StartToken > ttr.EndToken {
-			_, _ = fmt.Fprintf(&buf, "%d:%d,%d:%d", dht.Murmur3MinToken, ttr.EndToken, ttr.StartToken, dht.Murmur3MaxToken)
+			_, _ = fmt.Fprintf(&buf, "%d:%d,%d:%d", ttr.StartToken, dht.Murmur3MaxToken, dht.Murmur3MinToken, ttr.EndToken)
```

Comment:
Why the order is changed here ? What is the meaning of this change ?

Reply:
As described in 6624e060:
```
refactor(scyllaclient): make split range easier to detect

It's easier to look for a substring than regex match when checking if given token range string was created by splitting overlapping range.
```
This change just makes it easier to detect when a split token range is included in the repair job - which is important when writing tests validating that SM sends correct amount of token ranges per job.